src/index.html

```diff
@@ -11,6 +11,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    action=""
+    method="post">
```

Comment:
```suggestion
    <form
      action=""
      method="post"
    >
```
Форматування і відступи
Можна почитати тут
https://github.com/mate-academy/style-guides/blob/master/htmlcss.md#indentation